packages/webview/README.md

```diff
@@ -0,0 +1,315 @@
+# webview
+
```

Comment:
if we leave this as is, it may confuse people, because technically we ARE shipping C and GO interface opportunities from day 1 ... thoughts